src/initial-parameters.ts

```diff
@@ -1,14 +1,12 @@
 import os from 'os';
 import path from 'path';
-import { getEnvironmentVariables } from './env';
 import { readCommandLineArguments } from './command-line-arguments';
 import { readProject, Project } from './project';
 
 interface InitialParameters {
   project: Project;
   tempDirectoryPath: string;
   reset: boolean;
-  today: Date;
```

Comment:
There's no need to read the date from `TODAY` nor pass this down to anywhere, as a date isn't used anymore to calculate the version string. So it gets removed from everywhere.